app/src/test/java/org/simple/clinic/editpatient/deletepatient/DeletePatientUpdateTest.kt

```diff
@@ -0,0 +1,47 @@
+package org.simple.clinic.editpatient.deletepatient
+
+import com.spotify.mobius.test.NextMatchers.hasEffects
+import com.spotify.mobius.test.NextMatchers.hasNoModel
+import com.spotify.mobius.test.UpdateSpec
+import com.spotify.mobius.test.UpdateSpec.assertThatNext
+import org.junit.Test
+import org.simple.clinic.patient.DeletedReason
+import java.util.UUID
+
+class DeletePatientUpdateTest {
+
+  private val updateSpec = UpdateSpec(DeletePatientUpdate())
+  private val patientUuid = UUID.fromString("27c9456d-c8a6-4de9-bdd4-f249dff59de1")
+  private val patientName = "John Doe"
+  private val defaultModel = DeletePatientModel.default(
+      patientUuid = patientUuid
+  )
+
+  @Test
+  fun `show confirm delete dialog when selected patient delete reason is not died`() {
+    val model = defaultModel.patientNameLoaded(patientName)
+    val patientDeleteReason = PatientDeleteReason.Duplicate
+
+    updateSpec
+        .given(model)
+        .whenEvent(PatientDeleteReasonClicked(patientDeleteReason))
+        .then(assertThatNext(
+            hasNoModel(),
+            hasEffects(ShowConfirmDeleteDialog(patientName = patientName, deletedReason = DeletedReason.Duplicate) as DeletePatientEffect)
+        ))
+  }
+
+  @Test
+  fun `show confirm died dialog when selected patient delete reason is died`() {
```

Comment:
Can we rename the tests to be something like `when patient is being deleted because they have died, then the delete patient confirmation alert should be shown`?

Reply:
We show the delete patient confirmation dialog when the delete reason is not the patient is dead.